tests/monotouch-test/AVFoundation/AVAudioFormatTest.cs

```diff
@@ -13,6 +13,12 @@ namespace MonoTouchFixtures.AVFoundation {
 	[Preserve (AllMembers = true)]
 	public class AVAudioFormatTest {
 
+		[SetUp]
+		public void Setup ()
+		{
+			TestRuntime.AssertXcodeVersion (6, 0);
```

Comment:
Looks like you'll want to check for Xcode 6.1 for macOS, and Xcode 6.0 for iOS...

macOS 10.10 came with Xcode 6.1: https://github.com/xamarin/xamarin-macios/blob/f70e949a7d72ffbf8a3eefdfff8538d7e9df8538/tests/common/TestRuntime.cs#L508-L509

iOS 8 came with Xcode 6.0: https://github.com/xamarin/xamarin-macios/blob/f70e949a7d72ffbf8a3eefdfff8538d7e9df8538/tests/common/TestRuntime.cs#L491-L492

At this point the code would probably be more self-explanatory and simpler by having two explicit platform checks:

```csharp
TestRuntime.AssertiOSSystemVersion (8, 0, throwIfOtherPlatform: false);
TestRuntime.AssertMacSystemVersion (10, 10, throwIfOtherPlatform: false);
```

`AVAudioFormat` exists in all versions of tvOS and watchOS, so there should be no problem there.